src/components/ConnectToQuickbooksOnlineButton/index.native.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {useCallback, useRef, useState} from 'react';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import type {WebViewNavigation} from 'react-native-webview';
+import {WebView} from 'react-native-webview';
+import FullPageOfflineBlockingView from '@components/BlockingViews/FullPageOfflineBlockingView';
+import Button from '@components/Button';
+import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
+import Modal from '@components/Modal';
+import useLocalize from '@hooks/useLocalize';
+import {getQuickBooksOnlineSetupLink} from '@libs/actions/connections/QuickBooksOnline';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Session} from '@src/types/onyx';
+import type {ConnectToQuickbooksOnlineButtonProps} from './types';
+
+type ConnectToQuickbooksOnlineButtonOnyxProps = {
+    /** Session info for the currently logged in user. */
+    session: OnyxEntry<Session>;
+};
+
+const renderLoading = () => <FullScreenLoadingIndicator />;
+
+function ConnectToQuickbooksOnlineButton({policyID, session}: ConnectToQuickbooksOnlineButtonProps & ConnectToQuickbooksOnlineButtonOnyxProps) {
+    const [isWebViewOpen, setWebViewOpen] = useState<boolean>(false);
+    const [isQuickbooksOnlineReady, setIsQuickbooksOnlineReady] = useState<boolean>(false);
```

Comment:
```suggestion
    const [isWebViewOpen, setWebViewOpen] = useState(false);
    const [isQuickbooksOnlineReady, setIsQuickbooksOnlineReady] = useState(false);
```
Suggested change
    const [isWebViewOpen, setWebViewOpen] = useState<boolean>(false);
    const [isWebViewOpen, setWebViewOpen] = useState(false);
NAB. Prefer inferred types from initial value when possible